lib/hijack/wrap_session.js

```diff
@@ -194,7 +194,7 @@ function wrapMethodHanderForErrors(name, originalHandler, methodMap) {
 
 function cloneError(err) {
   if(err instanceof Meteor.Error) {
-    var newError = new Meteor.Error(err.error, err.reason);
+    var newError = err.clone();
```

Comment:
Is this a recently introduced API. Kadira package still can be used with even Meteor `0.7.2`. So, I need make sure this does not break there apps.
If this is available still on 0.7.2, we are good to go :)